test/blueprints/Do.Test.Blueprints.Service.Application/ConfigurationOverrider/ConfigurationOverriderExtensions.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Do.Architecture;
+using Do.Test.ConfigurationOverrider;
+
+namespace Do;
+
+public static class ConfigurationOverriderExtensions
+{
+    public static void AddConfigurationOverrider(this List<IFeature> source) => source.Add(new ConfigurationOverriderFeature());
+}
```

Comment:
document configuration overrider convention (both extension and feature) in `architecture/feature.md#conventions`